samples/bluetooth/hci_vs_scan_req/README.rst

```diff
@@ -1,13 +1,14 @@
-.. _bluetooth-hci-vs-scan-req-sample:
+.. zephyr:code-sample:: bluetooth_hci_vs_scan_req
+   :name: HCI Vendor-Specific Scan Request
+   :relevant-api: bluetooth
 
-Bluetooth: HCI VS Scan Request
-##############################
+   Manage vendor-specific HCI commands to obtain scan request events even using xxx.
```

Comment:
Use vendor-specific HCI commands to enable Scan Request events when using legacy advertising.